src/renderer/qssWidget/js/qssScreenCaptureWidget.js

```diff
@@ -0,0 +1,300 @@
+/**
+ * The QSS Screen Capture widget
+ *
+ * Shows a list of buttons that do a screen or video capture using the shareX command,
+ * its using siteconfig's shareXPath folder and the provided command from the button
+ *
+ * Copyright 2017 Raising the Floor - International
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ * The research leading to these results has received funding from the European Union's
+ * Seventh Framework Programme (FP7/2007-2013) under grant agreement no. 289016.
+ * You may obtain a copy of the License at
+ * https://github.com/GPII/universal/blob/master/LICENSE.txt
+ */
+
+"use strict";
+(function (fluid) {
+    var gpii = fluid.registerNamespace("gpii");
+
+    /**
+     * Represents the QSS Screen Capture widget.
+     */
+    fluid.defaults("gpii.qssWidget.screenCapture", {
+        gradeNames: ["fluid.viewComponent", "gpii.psp.heightObservable", "gpii.psp.selectorsTextRenderer"],
+        model: {
```

Comment:
Document in your members that there is a volatile member used for the keyboard event - I recommend something like
````
members: {
    // Holds the last keyboard event received during the "close" operation
    closingKeyboardEvent: null
    }
````

Reply:
Done